src/lib/Scenes/Artwork/Components/ContextCard.tsx

```diff
@@ -138,7 +138,7 @@ export class ContextCard extends React.Component<ContextCardProps, ContextCardSt
           header = context.isAuction ? "In auction" : "In sale"
           name = context.name
           href = context.href
-          meta = context.formattedStartDateTime
+          meta = context.isLiveOpen ? "In progress" : context.formattedStartDateTime
```

Comment:
Ah! Thanks for linking. I think the current behavior is fine 👍.

It would be nice to consolidate the logic somewhere but I'm not sure if that MP field is the right place.